connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerder.java

```diff
@@ -206,14 +206,17 @@ public synchronized void restartTask(ConnectorTaskId taskId, Callback<Void> cb)
         if (!configState.contains(taskId.connector()))
             cb.onCompletion(new NotFoundException("Connector " + taskId.connector() + " not found", null), null);
 
-        Map<String, String> taskConfig = configState.taskConfig(taskId);
-        if (taskConfig == null)
+        Map<String, String> taskConfigs = configState.taskConfig(taskId);
```

Comment:
would `taskConfigProperties` be a better name as those get used to create the `taskConfig` object?